tests/ignite/contrib/handlers/test_lr_finder.py

```diff
@@ -462,4 +463,130 @@ def test_plot_multiple_param_groups(
     assert ax.get_xlabel() == "Learning rate"
     assert ax.get_ylabel() == "Loss"
     ax.figure.savefig("dummy_muliple_param_groups2.jpg")
-    assert path.exists("dummy_muliple_param_groups2.jpg")
+    assert os.path.exists("dummy_muliple_param_groups2.jpg")
+
+
+def _test_distrib_log_lr_and_loss(device):
+    from ignite.handlers import PiecewiseLinear
+
+    class DummyModel(nn.Module):
+        def __init__(self, n_channels=10, out_channels=10, flatten_input=False):
+            super(DummyModel, self).__init__()
+
+            self.net = nn.Sequential(
+                nn.Flatten() if flatten_input else nn.Identity(), nn.Linear(n_channels, out_channels)
+            )
+
+        def forward(self, x):
+            return self.net(x)
+
+    model = DummyModel()
+    optimizer = SGD(model.parameters(), lr=1e-4, momentum=0.0)
+    lr_finder = FastaiLRFinder()
+
+    # minimal setup for lr_finder to make _log_lr_and_loss work
+    rank = idist.get_rank()
+    loss = 0.01 * (rank + 1)
+
+    _lr_schedule = PiecewiseLinear(optimizer, param_name="lr", milestones_values=[(0, 1e-4), (1, 10)])
```

Comment:
Let's see if we could use `unittest.mock` instead ? Such that we could remove model and optimizer from this test.

Reply:
How about mocking `_lr_schedule` ?